### DISCUSSION TITLE
what does include_match_in_root do?

### DISCUSSION THREAD

**Discussion by sucremad** [jertel/elastalert2#1357] [Q&A] [answered]
Hi,

I could not see `include_match_in_root ` in the documentation. What is it for? could you please explain?

**Comment by jertel** [comment, marked as answer]
It adds all the matched record key/value pairs into the alert body content, which is usually the main body of an alert, like a Slack message, or email body.

**Reply by nsano-rururu** [reply]
A note about include_match_in_root. Items whose first character is `_` are not applicable.

https://github.com/jertel/elastalert2/blob/93f85108a3bc9c48c8542a04b6214dda17dc1c62/elastalert/elastalert.py#L1437-L1438
